<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="Torngrafik" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Inställningar</string>
    </property>
    <widget class="QMenu" name="menuSize">
     <property name="title">
      <string>Storlek</string>
     </property>
     <addaction name="action3"/>
     <addaction name="action5"/>
     <addaction name="action10"/>
     <addaction name="action20"/>
     <addaction name="action50"/>
     <addaction name="action100"/>
    </widget>
    <widget class="QMenu" name="menuSpeed">
     <property name="title">
      <string>hastighet</string>
     </property>
     <addaction name="actionSlow"/>
     <addaction name="actionNormal"/>
     <addaction name="actionFast"/>
    </widget>
    <addaction name="actionReset"/>
    <addaction name="menuSize"/>
    <addaction name="actionMix"/>
    <addaction name="menuSpeed"/>
   </widget>
   <widget class="QMenu" name="menuUppdrag">
    <property name="title">
     <string>Uppdrag</string>
    </property>
    <addaction name="actionmoveDiscs"/>
    <addaction name="actionsolveMixed"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuUppdrag"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSlow"/>
   <addaction name="actionNormal"/>
   <addaction name="actionFast"/>
   <addaction name="separator"/>
   <addaction name="actionReset"/>
   <addaction name="actionmoveDiscs"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/reset.png</normaloff>:/images/reset.png</iconset>
   </property>
   <property name="text">
    <string>Återställ</string>
   </property>
  </action>
  <action name="action3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="action10">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </action>
  <action name="action20">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </action>
  <action name="action50">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </action>
  <action name="action100">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </action>
  <action name="actionMix">
   <property name="text">
    <string>Blanda</string>
   </property>
  </action>
  <action name="actionSlow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/turtle.png</normaloff>:/images/turtle.png</iconset>
   </property>
   <property name="text">
    <string>långsam</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/rabbit.png</normaloff>:/images/rabbit.png</iconset>
   </property>
   <property name="text">
    <string>lagom</string>
   </property>
  </action>
  <action name="actionFast">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/rocket.png</normaloff>:/images/rocket.png</iconset>
   </property>
   <property name="text">
    <string>snabb</string>
   </property>
  </action>
  <action name="actionmoveDiscs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/play1.png</normaloff>:/images/play1.png</iconset>
   </property>
   <property name="text">
    <string>moveDiscs</string>
   </property>
  </action>
  <action name="actionsolveMixed">
   <property name="text">
    <string>solveMixed</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Torngrafik</class>
   <extends>QWidget</extends>
   <header>torngrafik.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
